plugins/mac-virtualcam/src/dal-plugin/CMakeLists.txt

```diff
@@ -32,38 +23,45 @@ target_sources(
           Defines.h
           Logging.h)
 
-if(NOT XCODE)
-  target_compile_options(obs-dal-plugin PRIVATE -fobjc-arc -fobjc-weak)
-endif()
+target_compile_options(obs-dal-plugin PRIVATE -fobjc-arc -fobjc-weak)
```

Comment:
Does Xcode now require this as well? This looks backwards to the other changes to remove Ninja CMake paths.

Reply:
Nope, that looks more like an oversight.